.gitignore

```diff
@@ -22,28 +22,27 @@
 *.uvoptx
 *.__i
 *.i
-*.txt
```

Comment:
I had to remove this gitignore entry due to it blocking staging of CMake files. Since it might be there for a reason, I wanted to give you heads up if you want to resolve it somehow with a regex or something.